src/renderer/screens/Assets/Assets.tsx

```diff
@@ -58,18 +58,22 @@ export const Assets = () => {
     setActiveAccounts(result);
   };
 
+  const availableConnectionsAmount = Object.values(connections).filter(
+    (c) => c.connection.connectionStatus === ConnectionStatus.CONNECTED,
+  ).length;
+
   const sortedChains = useMemo(
     () =>
       sortChains(
         Object.values(connections).filter((c) => {
           const isDisabled = c.connection.connectionType === ConnectionType.DISABLED;
           const hasMultisigAccount = activeAccounts.some(isMultisig);
-          const hasMultiPallet = !hasMultisigAccount || Boolean(c.api?.tx.multisig);
+          const hasMultiPallet = !hasMultisigAccount || c.connection.hasMultisigPallet !== false;
```

Comment:
`c.connection.hasMultisigPallet !== false;` should be `true` or `undefined`?

Reply:
right, it is for the first loading